README.md

```diff
@@ -23,6 +27,8 @@ KAPEL is container-native APEL accounting for Kubernetes.
   - For large production deployments (examples are based on a cluster with about 125 nodes and 7000 cores):
     - Increase `.Values.prometheus.querySpec.timeout` (e.g. ~ 1800s) to allow long queries to succeed.
     - Apply sufficient CPU and memory resource requests and limits.
+  - If Prometheus is configured to require authentication, configure `.Values.processor.prometheus_auth` to specify an appropriate kubernetes
```

Comment:
This section is meant to document the external requirements, i.e. things you need before you start configuring KAPEL. Values like this one for configuring KAPEL itself are documented in the chart docs.